tools/crm_node.c

```diff
@@ -194,6 +235,57 @@ node_name_xml(pcmk__output_t *out, va_list args) {
     return pcmk_rc_ok;
 }
 
+PCMK__OUTPUT_ARGS("partition-list", "GList *")
+static int
+partition_list_default(pcmk__output_t *out, va_list args)
+{
+    GList *nodes = va_arg(args, GList *);
+
+    GString *buffer = NULL;
+
+    for (GList *node_iter = nodes; node_iter != NULL; node_iter = node_iter->next) {
+        pcmk_controld_api_node_t *node = node_iter->data;
+        if (pcmk__str_eq(pcmk__s(node->state, ""), "member", pcmk__str_none)) {
```

Comment:
FYI don't need `pcmk__s()` with `pcmk__str_eq()` since it handles NULL

Reply:
Fixed.